### PR TITLE
Feature/dcwc 1421 add accessiblity tests 1

### PR DESCRIPTION
**Pull request by nabil386**
### Description
add accessibility tests and fix duplicate ids
List of proposed changes:

- add accessibility test for dashboard
- add accessibility test for HorizontalRule
- fix duplicate ids

### CODE REVIEW

File: components/atoms/BenefitCardHeaderActive.js [hunk 1]
```diff
@@ -18,17 +18,20 @@ export default function BenefitCardHeaderActive(props) {
         </h2>
       </div>
       <div className="mx-auto sm:grid sm:grid-cols-4 sm:divide-x-2">
-        <div id="benefitName" className="col-span-1 py-4 md:px-0 lg:px-3">
+        <div
+          id={props.benefit.benefitType + '-active'}
```
Comment (arsanig) [inline components/atoms/BenefitCardHeaderActive.js:22]: What's the goal with this string concatenation in the different files?

**Review by amirpourpanahi — APPROVED** [review]

**Review by arsanig — APPROVED** [review]
Good for now. Should be revisited on connection with APIs. Hopefully there may be a way around the hardcoded string literals
